lightningd/dual_open_control.c

```diff
@@ -1662,6 +1662,54 @@ static void handle_validate_rbf(struct subd *dualopend,
 	tal_free(set_psbt);
 }
 
+static struct command_result *
+json_openchannel_bump(struct command *cmd,
+		      const char *buffer,
+		      const jsmntok_t *obj UNNEEDED,
+		      const jsmntok_t *params)
+{
+	struct channel_id *cid;
+	struct channel *channel;
+	struct amount_sat *amount;
+	struct wally_psbt *psbt;
+	struct open_attempt *oa;
+
+	if (!param(cmd, buffer, params,
+		   p_req("channel_id", param_channel_id, &cid),
+		   p_req("amount", param_sat, &amount),
+		   p_req("initialpsbt", param_psbt, &psbt),
```

Comment:
Um, what is "amount" here?  That's previously been used for "amount to open the channel with", so is this the same?

(I guess I'll find out when I get to the docs?)

Reply:
As proposed, the spec allows you to update the total funding amount on an RBF attempt -- it's the only thing you really can change between attempts, all other channel parameters remain the same between the original `open_channel2` message and `init_rbf`.